Checkpoint3/Program.cs

```diff
@@ -0,0 +1,236 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.EntityFrameworkCore;
+
+namespace Checkpoint3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            //initialises controller class and calls Run method
+            Controller controller = new Controller();
+            controller.Run(); 
+        }
+    }
+    public class Controller
+    {   
+        //instantiates Dao
+        static DAO theDao = new DAO();
+        //sets itemList to Dao list
+        static List<Item> itemList = theDao.list();
```

Comment:
I would not make the list a static variable here. the point of the dao is to get you the ilst when needed. since it might change in the database outside of this program.